ceno_zkvm/src/scheme/mock_prover.rs

```diff
@@ -1011,7 +1011,7 @@ Hints:
                         let mut records = vec![];
                         for (row, record_rlc) in write_rlc_records.into_iter().enumerate() {
                             // TODO: report error
-                            assert_eq!(writes.insert(record_rlc), true);
+                            writes.insert(record_rlc);
```

Comment:
This is supposed to later collect and return the errors (not assert), right?

Reply:
This assertion should not be removed. I've restored it.